### PR TITLE
Add gravatar images

### PR DESCRIPTION
**Pull request by NathanielWroblewski**
- Move check-ins above feedback
- Pull javascript out into js files
- Add a css file to style an on-hover tooltip containing the coach email
- Pull gravatars instead of just emails
- Add a function that allows MD5 hashing of emails required for pulling gravatar images
- Make it looks sexy mmm
  screen shot 2014-11-03 at 10 19 00 pm

### CODE REVIEW

**Comment by amgando** [comment]
so much codez! md5. pffft. 

here's something you might enjoy from @mattbaker, a Chicago teacher who ships software over breakfast

http://stackoverflow.com/a/1655795

**Comment by jonathanpeterwu** [comment]
this looks awesome. I'll QA it real quick then put it into production. @SamSamskies  suggested thinking about turning this into a backbone app thoughts?

File: index.html [hunk 1]
```diff
@@ -74,28 +80,6 @@
         </script>
 
      <div><hr><p><a href="https://podio.com/webforms/9873101/716953" target="_blank">Submit your feedback here</a></p></div>
-
-        <script>
-          $(document).ready(function() {
-            var database = new Firebase('https://dbc-coaches.firebaseio.com/')
-              , checkIns = database.child('checkin')
-
-            checkIns.on('value', checkIn)
```
Comment (jonathanpeterwu) [inline index.html:83]: thanks for pulling this soupy mess out of the code!

**Comment by amgando** [comment]
re: backbone -- i don't personally feel like the site needs an overhaul right now.  it's a simple site, easy to edit, easy to throw away.  i'm concerned that if we start investing in making it "better" then suddenly we end up with something that takes us more time to change and maintain.  in fact we don't even know what to do with this thing yet. we literally just set it up.

my vote is "no" on an overhaul.

**Comment by SamSamskies** [comment]
Alright, I have a really simple solution to DRY up the code into one page
without using a framework. I'll make a PR later. The 5 pages of repeated
code is making me nauseous.

Sam

**Comment by jonathanpeterwu** [comment]
@amgando - i agree with the reservations as well, just wanted to loop the convo in
@SamSamskies - tag us in the PR and we can take a look at it.

**Comment by mattbaker** [comment]
Side-note/tangent/"FWIW", the first test I have on "should it be backbone" is "what value does a single page app add to the user's experience?"

"Feeling faster" is a common reason, but isn't usually enough for me to justify it. For me, it has to be a product that really benefits from the single-page design. Not navigating away from the page as I click items in instacart, not reloading the page every time I send a message in a chat room, etc.

**Comment by SamSamskies** [comment]
There is duplicate code on every single HTML page is the biggest reason I
wanted to Backbonify it. And it would be really easy and quick to do. In
the mean time, I came up with a simpler solution to consolidate all HTML
pages into one. I'll finish the PR when I get home. If you guys enjoy copy
pasting code into 5 different files every time you want to make an update
to that sidebar, feel free to trash my PR.

Sam

On Tue, Nov 4, 2014 at 8:48 AM, Johnny Wu notifications@github.com wrote:

> this looks awesome. I'll QA it real quick then put it into production.
> @SamSamskies https://github.com/SamSamskies suggested thinking about
> turning this into a backbone app thoughts?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/dbc-coaches/public/pull/6#issuecomment-61671690.

## 

You are awesome. :)
http://samsamskies.github.io

**Comment by jonathanpeterwu** [comment]
no stress. both valid points. I like your PR approach. mainly just routing to render sections, no real backbone included

**Pull request merged**